Packs/CaseManagement-Generic/Scripts/LinkIncidentsButton/LinkIncidentsButton.py

```diff
@@ -0,0 +1,15 @@
+import demistomock as demisto
+from CommonServerPython import *
+
+linkedIncidentIDs = demisto.args().get("linkedIncidentIDs")
+action = demisto.args().get("action")
+if action not in ["link", "unlink"]:
+    action = "link"
+counter = int(demisto.args().get("retryLimit"))
+res = demisto.executeCommand("linkIncidents", {"linkedIncidentIDs": linkedIncidentIDs, "action": action})
+while isError(res[0]) and counter > 0 and ("DB Version" in res[0]["Contents"]):
+    time.sleep(2)
```

Comment:
![image](https://user-images.githubusercontent.com/53565845/92321841-e5597700-f035-11ea-8680-819e65af120d.png)
`time.sleep` statement is still present, we need to remove it as @ShahafBenYakir said.